h/static/styles/partials-v2/_search.scss

```diff
@@ -117,18 +114,20 @@
 }
 
 .search-result-bucket__annotation-cards {
-  flex-grow: 1;
   max-width: 500px;
+  flex-shrink: 0;
   padding-left: 0;
   padding-right: 0;
 }
 
 // Card displaying stats about a group of annotations in search results
 .search-bucket-stats {
   @include font-normal;
-  width: 210px;
   margin-left: 30px;
   word-wrap: break-word;
```

Comment:
My preferred solution would be to let our CSS building tooling deal with mapping standardized property names to non-standard ones, as we do with properties that need vendor prefixing. Looks like this was requested upstream in autoprefixer but not actually implemented: https://github.com/postcss/autoprefixer/issues/558

A PR that either converts all uses of `overflow-wrap` to `word-wrap` would be OK with me though - since realistically support for the legacy name is never going to be removed from the web platform.